just like the other part it's already patched in, not actually used
--
diff --git a/src/shared/user-record-nss.h b/src/shared/user-record-nss.h
index 7a41be7..3a970a6 100644
--- a/src/shared/user-record-nss.h
+++ b/src/shared/user-record-nss.h
@@ -2,7 +2,6 @@
 #pragma once
 
 #include <grp.h>
-#include <gshadow.h>
 #include <pwd.h>
 #include <shadow.h>
 
